libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html

```diff
@@ -4,5 +4,4 @@
       <elewa-group-social-impact-hero-section></elewa-group-social-impact-hero-section>
```

Comment:
Where is the component you created ??

Reply:
This is the name the component; SocialImpactCooperativeComponent, it reuses the component created in #31, and basically passes inputs to that component